### PR TITLE
Unmapped resource bug logging changes and service account issue fixed

### PR DESCRIPTION
**Pull request by arvindsingh-cloud**
## Description
</br>
 Brief description of the requested change.

## Checklist
- [ ] I have read the instructions mentioned in [Contribute to Code](/Contributing.md).
- [ ] I have read and understood the criteria described under [submitting changes](/Contributing.md#submitting-changes). 
- [ ] The title of the PR clearly describes the intent of the PR.
- [ ] This PR does not introduce any breaking changes to the code.

### CODE REVIEW

File: src/AzSK.ADO/Framework/BugLog/BugMetaInfoProvider.ps1 [hunk 1]
```diff
@@ -141,13 +150,14 @@ class BugMetaInfoProvider {
...
                                 $apiURL = "https://dev.azure.com/{0}/_apis/securityroles/scopes/distributedtask.serviceendpointrole/roleassignments/resources/{1}_{2}" -f $organizationName, $projectId, $ControlResult.ResourceContext.ResourceId.split('/')[-1]
                                 $responseObj = [WebRequestHelper]::InvokeGetWebRequest($apiURL);
-                                $roles =   @($responseObj | where {$_.role.displayname -eq 'Administrator'} |select identity) | where {(-not ($_.identity.displayname).Contains("\")) -and ($_.identity.displayname -notin @("GitHub", "Microsoft.VisualStudio.Services.TFS"))}
+                                $roles =   @(($responseObj | where {$_.role.displayname -eq 'Administrator'} |select identity) | where {(-not ($_.identity.displayname).Contains("\")) -and ($_.identity.displayname -notin @("GitHub", "Microsoft.VisualStudio.Services.TFS"))} )
                                 if ($roles.Count -gt 0) {
                                     $userId = $roles[0].identity.id
                                     $assignee = $this.getUserFromUserId($organizationName, $userId)
```
Comment (juhi037) [inline src/AzSK.ADO/Framework/BugLog/BugMetaInfoProvider.ps1:163]: should we check in getUserFromUserId at the end if user is active?

Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/BugLog/BugMetaInfoProvider.ps1:163]: Checking latter on outside in Autobuglog class

**Review by juhi037 — APPROVED** [review]

File: src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1 [hunk 2]
```diff
@@ -105,9 +105,12 @@ class AutoBugLog : EventBase  {
                 else {
                     $resourceOwner = $AssignedTo;
                 }
+
+                #check for user is active or not
+                $isUserActive = $metaProviderObj.isUserActive($this.OrganizationName, $AssignedTo);
```
Comment (SaikumarRagala) [inline src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1:110]: Should it be $resourceOwner instead of $AssignedTo?

Comment (arvindsingh-cloud) [reply src/AzSK.ADO/Framework/BugLog/AutoBugLog.ps1:110]: no resource owner is just to display in bug description, we log bug for assignee

**Review by SaikumarRagala — APPROVED** [review]

**Pull request merged**